### ISSUE TITLE
problem with background images in css

### ISSUE THREAD

**Issue by nimasdj** [symfony/AsseticBundle#338]
with symphony framework I did dump assets assets:install.
css file is hard copied to /web/bundles/appbundle/css/style.css I guess for background image in css I should have a relative path to reach outside of /web/ folder like this?

    background-image: url(../../../../bundles/appbundle/images/top_bg.jpg);

but it doesn't work yet, I have filter='cssrewrite' in css tag too. probably I have to add that I am only editing the css file located at the path above after assets install, I did not edit the one in /Acme/Bundle/Resources/public/css any more. Then I did run assets:dump, now in /web/ folder there are two folders for images and css, I looked at new css and see the path became like this:

    background-image: url(../../bundles/applicationadmin/images/top_bg.jpg);

But still all images are broken. I searched stackoverflow and found [this question][1], but still have problem. what else should I do?

please advice.

  [1]: http://stackoverflow.com/questions/9500573/path-of-assets-in-css-files-in-symfony-2

**Comment by xabbuh** [comment]
Isn't this the same as #336?

**Comment by nimasdj** [comment]
Actually that issue was about images in general not only in css. I even did not know how to show an image in twig. With the help of that guy I was able to dump assets and show the images and load css. In this issue I meant to ask background images in css are broken. Please help.

**Comment by stof** [comment]
Are you using the ``@Bundle`` notation to reference your CSS file ? This is not compatible with the cssrewrite filter.

**Comment by nimasdj** [comment]
@stof No. I am using 'bundles/appbundle/css/*' and everything works fine if I use /web/app.php, but if I move app.php outside of /web/ folder to root and apply chsnges in include paths. Css background images will be broken. Should I use
read_from = "web"
in assetic cinfiguration to fix this?
